sechub-pds-solutions/multi/docker/scripts/njsscan.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+# SPDX-License-Identifier: MIT
+
+njsscan --sarif --output "$PDS_JOB_RESULT_FILE" "$PDS_JOB_SOURCECODE_UNZIPPED_FOLDER/"
+
+exit 0
```

Comment:
This forces exit code 0. (See my comment above)
Is this wanted?

Reply:
Yes, see my comment above: https://github.com/Daimler/sechub/pull/941#discussion_r785023745.